site/public/js/drag-and-drop.js

```diff
@@ -91,6 +91,34 @@ function dropWithMultipleZips(e){
     }
 }
 
+// show progressbar when uploading files 
+function progress(e){
+    let loading_bar = $("#loading_bar");
+    let loading_bar_element = $("#loading_bar_element");
+    let loading_bar_percentage = $("#loading_bar_percentage");
+
+    if(!(loading_bar_element && loading_bar && loading_bar_percentage)){
+        return false;
+    }
+
+    if(loading_bar.css("display", "none")){
+        $(loading_bar).fadeIn(10);
+    }
+
+    if(e.lengthComputable){
+        let maximum = e.total;
+        let current_state = e.loaded;
+        let perc = (current_state * 100)/maximum;
+
+        loading_bar_element.css("width", perc.toFixed(2) + "%");
```

Comment:
See comment above on how to update progress bar

Reply:
sure, as well.